### ISSUE TITLE
documentation: describe when to and when not to use sveltekit

### ISSUE THREAD

**Issue by zahachtah** [sveltejs/kit#2174]
### Describe the problem

My example: I am going to build a spa, not server rendered, and all pages dynamic (i.e. no routing, except possibly by document-id from documents loaded from a couchdb server).

Should I use sveltekit anyways instead of plain svelte? I heard sveltekit could/should become default for any svelte project. And if I remember correctly Rich released a component (knobby?) made with sveltekit. 

It seems using vite for developing is one argument for always using sveltekit. 

will the REPL on the homepage eventually use sveltekit?

tl;dr: find it difficult to decide if I should use sveltekit since I won't use most of the "features" advertised but on the other hand I hear it may become the default development platform even for components.

### Describe the proposed solution

could there be a short section in the documentation when NOT to use sveltekit?

### Importance

nice to have

**Comment by benmccann** [comment]
You don't have to use all the features. You can just use the ones that are relevant to you. I'm not sure what we would add as a usecase of when not to use SvelteKit. Happy to consider if there are suggestions, but this may end up getting closed if there are not actionable suggestions

> I hear it may become the default development platform even for components

There is `svelte-kit package` that is meant to address this use case

**Comment by zahachtah** [comment]
Ok! so this answers it for me: sveltekit can (should?) be used for all future projects. So my suggestion then is just to make it somewhat visible in the documentation that sveltekit is very suitable for small scale projects also and the potential benefits (using vite + ? ) even if the advertised features (e.g. SSR, static pages router) and are turned off. This project, and svelte in general, are amazing, thanks to all who put so much effort into it!
